samples/javascript_nodejs/25.logger-bot/README.md

```diff
@@ -0,0 +1,69 @@
+
+This bot has been created using [Microsoft Bot Framework][10], it shows how to create a simple echo bot with state. The bot maintains a simple counter that increases with each message from the user. This bot example uses [`restify`][1].
+
+# To run the bot
+- Install modules and start the bot
+    ```bash
+    npm i & npm start
+    ```
+    Alternatively you can also use nodemon via
+    ```bash
+    npm i & npm run watch
+    ```
+
+# Testing the bot using Bot Framework Emulator
+[Microsoft Bot Framework Emulator][2] is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+- Install the Bot Framework emulator from [here][3]
+
+## Connect to bot using Bot Framework Emulator **V4**
+- Launch Bot Framework Emulator
+- File -> Open Bot Configuration
+- Select `loggerBot.bot` file
+
+# Bot state
+A key to good bot design is to track the context of a conversation, so that your bot remembers things like the answers to previous questions. Depending on what your bot is used for, you may even need to keep track of conversation state or store user related information for longer than the lifetime of one given conversation.
+
+In this example, the bot's state is used to track number of messages.
+
+ A bot's state is information it remembers in order to respond appropriately to incoming messages. The Bot Builder SDK provides classes for [storing and retrieving state data][4] as an object associated with a user or a conversation.
+
+    - Conversation properties help your bot keep track of the current conversation the bot is having with the user. If your bot needs to complete a sequence of steps or switch between conversation topics, you can use conversation properties to manage steps in a sequence or track the current topic. Since conversation properties reflect the state of the current conversation, you typically clear them at the end of a session, when the bot receives an end of conversation activity.
+
+    - User properties can be used for many purposes, such as determining where the user's prior conversation left off or simply greeting a returning user by name. If you store a user's preferences, you can use that information to customize the conversation the next time you chat. For example, you might alert the user to a news article about a topic that interests her, or alert a user when an appointment becomes available. You should clear them if the bot receives a delete user data activity.
+
+# Deploy this bot to Azure
+You can use the [MSBot][5] Bot Builder CLI tool to clone and configure the services this sample depends on.
+
+To install all Bot Builder tools -
+
+Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher
+
+```bash
+npm i -g msbot chatdown ludown qnamaker luis-apis botdispatch luisgen
+```
+
+To clone this bot, run
+```
+msbot clone services -f deploymentScripts/msbotClone -n myChatBot -l <Azure-location> --subscriptionId <Azure-subscription-id>
```

Comment:
This one is the same in all sample bot readmes, so it might be better to open an issue/pr for that change specifically so it isn't just this one getting the fix.